airbyte-integrations/connectors/source-metabase/setup.py

```diff
@@ -5,7 +5,7 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk~=0.2", "requests>=2.28.0", "types-requests>=2.27.30"]
+MAIN_REQUIREMENTS = ["airbyte-cdk", "requests>=2.28.0", "types-requests>=2.27.30", "cachetools"]
```

Comment:
we shouldn't need this dependency here since it'll be included in the cdk as part of https://github.com/airbytehq/airbyte/pull/19716

Reply:
thanks, removed